git_machete/cli.py

```diff
@@ -732,9 +751,11 @@ def strip_remote_name(remote_branch: RemoteBranchShortName) -> LocalBranchShortN
             machete_client.slide_out(
                 branches_to_slide_out=list(map(LocalBranchShortName.of, branches)),
                 opt_down_fork_point=cli_opts.opt_down_fork_point,
+                opt_yes=cli_opts.opt_yes,
```

Comment:
Hmmm `--yes` is NOT supported (and probably shouldn't be supported) for `slide-out` right?

Reply:
Yes, it shouldn't